pkgs/applications/networking/ids/zeek/broker/caf-incubator.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, caf
+, openssl
+}:
+stdenv.mkDerivation {
+  pname = "caf-incubator";
+  version = "2022-09-11-zeek";
+
+  src = fetchFromGitHub {
+    owner = "actor-framework";
+    repo = "incubator";
+    rev = "cdfadf40038e6139117190f83be27553e0a93317";
+    hash = "sha256-RO+ctRLp8hhii28ptMi7nhh77JBSZxShuennSD3Wbk4=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ caf ];
+
+  cmakeFlags = [
+    "-DCAF_INC_ENABLE_BB_MODULE=OFF"
+    "-DCAF_INC_ENABLE_EXAMPLES=OFF"
+    "-DCAF_INC_ENABLE_TESTING=OFF"
```

Comment:
Why not tests?

Reply:
This file is a leftover from an earlier version of this PR. I'll remove.